NEWS

```diff
@@ -13,6 +13,14 @@ CHANGES WITH 256 in spe:
           section, then all assigned VLAN IDs on the interface that are not
           configured in the .network file are removed.
 
+        * systemd-logind now always enforces inhibitor locks, regardless of the
+          uid of the caller, even if root. This is not a security restriction,
+          as a privileged user can always list the active inhibitor locks using
+          'systemd-inhibit', but it is a safety measure, to avoid accidental
+          reboot/shutdown/etc that were not meant to happen. Previously,
+          requests from root or the same uid as the lock holder were always
+          bypassing inhibitors by default.
```

Comment:
This sounds a bit off. It's not listing of the inhibitors that is really the important operation.

Maybe:
> systemd-logind now always obeys inibitor locks, where previously it ignore locks taken by the caller or when the caller was root. A privileged caller can always remove close the other sessions, remove the inhibitor locks, or use `--force` to ignore the inhibitors. This change thus doesn't affect security, since everything that was possible before is still possible, but should make the inhibitor logic easier to use and understand, helping avoid accidetnal reboots and shutdowns.